docs/code-style.md

```diff
@@ -0,0 +1,667 @@
+Disciplina Style Guide
+============================
+> This style guide is mostly a copy of
+> [Serokell Haskell Style Guide](https://github.com/serokell/serokell-util/blob/master/serokell-style.md)
+> with some modifications.
+> This style guide's aims are code beauty, readability and understandability.
+
+You can find our other formatting utilites and guidelines which expand the code style:
+* [.stylish-haskell.yaml](.stylish-haskell.yaml)
+* [Universum](https://github.com/serokell/universum)
+* [Custom _HLint_ rules](https://github.com/input-output-hk/cardano-sl/blob/master/HLint.hs)
+
+General guide lines
+-------------------
+### Line Length
+
+Maximum line length is *80 characters* or *100 characters* if necessary.
+
+Modern screens have high definition and big width.
+But with some tiling managers and two terminals on one screen you are not able to
+see many characters on one line.
+On the other hand, restricting line size to a very small number like 80 leads to
+some crazy indentation despite the fact that
+shorter lines should force you to write well structured code.
+That's why *100* is a reasonable compromize.
+
+
+### Indentation
+
+Tabs are illegal.  Use spaces for indenting.  Indent your code blocks
```

Comment:
Did you insert extra spaces after each sentence to emphasize restriction on using spaces? :peka:

Reply:
I believe that this is even older than main Serokell styleguide. Double space after a period was some kind of standard for written plaintext, even `gtypist` enforces that in its exercises.

Reply:
Oh lol, I got it. Hilarious